access-modifier-checker/src/it/final/caller/src/main/java/caller/Caller.java

```diff
@@ -0,0 +1,12 @@
+package caller;
+
+import api.Api;
+
+public class Caller {
+
+    static void x() {
+        Api x = null;
+        x.notReallyFinal(); // should not fail
```

Comment:
What magic is this?

Reply:
just checking that callers can still use the api. We just want a method call in the bytecode